scheduler/scheduler-smartproxy/src/main/java/org/ow2/proactive/scheduler/smartproxy/SmartProxyImpl.java

```diff
@@ -701,4 +717,26 @@ public Boolean call() throws Exception {
         }
     }
 
+    @Override
+    public Page<TaskId> getTaskIds(String taskTag, long from, long to,
+            boolean mytasks, boolean running, boolean pending, boolean finished, int offset, int limit)
+                    throws NotConnectedException, PermissionException {
+        return schedulerProxy.getTaskIds(taskTag, from, to, mytasks, running, pending, finished,
+                offset, limit);
+    }
+
+    @Override
+    public Page<TaskState> getTaskStates(String taskTag, long from, long to,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)